modules/core/05-port/keeper/keeper.go

```diff
@@ -81,3 +81,9 @@ func (k *Keeper) LookupModuleByPort(ctx sdk.Context, portID string) (string, *ca
 
 	return types.GetModuleOwner(modules), capability, nil
 }
+
+// GetRoute returns a IBCModule for a given module, and a boolean indicating
+// whether or not the route is present.
+func (k *Keeper) GetRoute(module string) (types.IBCModule, bool) {
```

Comment:
we have another router in 02-client for light client modules and the func signature we use there is `Route(clientID string) (exported.LightClientModule, bool)`

it would be nice to have some symmetry between the two imo!

Reply:
Sure that makes sense! Done